x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/index.tsx

```diff
@@ -490,6 +504,38 @@ export const CreatePackagePolicySinglePage: CreatePackagePolicyParams = ({
             <EuiSpacer size="xl" />
           </>
         ) : null}
+        {showSecretsDisabledCallout && (
```

Comment:
Should we display this in the multiple page layout too? may not be relevant as I think we only display this on cloud for first time Fleet user so probably with a recent Fleet Server

Reply:
Yeah I don't think this necessarily makes sense on the multi-page layout. I'll take a look though - maybe we can place the same callout in the inline policy form that renders there.

Reply:
This PR has gone on for a while so I think let's just punt on the multipage layout. It's only used for now in first-time cloud onboarding so don't think it's super relevant.